code/modules/reagents/bonsai.dm

```diff
@@ -9,8 +9,9 @@
 	name = "Laurelin bonsai"
 	desc = "A small tree, gifted to the club by a previous patron. It subsists off of numerous alcohols, and produces fruits and vegetables in return."
 
-	icon = 'icons/obj/plants.dmi'
-	icon_state = "plant-21" //Placeholder until we can get a proper sprite for them.
+	icon = 'icons/obj/bonsai.dmi'
+	icon_state = "bonsai_1"
+	layer = ABOVE_MOB_LAYER
```

Comment:
So this means someone can hide under the sprite. **No** please.

Reply:
it looks dumb when above it

Reply:
changed, after checking, being above the bonsai layer is ok